packages/terra-consumer-nav/src/components/modal/Modal.jsx

```diff
@@ -14,50 +15,44 @@ const propTypes = {
    */
   title: PropTypes.string,
   /**
-   * Content to be rendered in the body of the dialog box.
-   */
-  content: PropTypes.element,
```

Comment:
is most of this just an attempt to align us with the terra modal more closely?
https://github.com/cerner/terra-core/blob/master/packages/terra-modal/src/Modal.jsx